csv-import-export.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Bring (and take) your own data with CSV import and export'
+description: 'Simplified data import and export in the Xata UI to bring, manage, and retrieve your own data'
```

Comment:
small nit, but I think we refer the the "Xata UI" as as the "Xata app" or in this case, we could simply say "in Xata".

Reply:
Ah, this was discussed a long long time ago and we decided to go with Xata UI (long enough that I no longer find the discussion), so I'm using it since. Agree that in "in Xata" would work here, though.

Reply:
ah, you're right. I thought we landed on Xata app for some reason. Too early for me I guess 😉 https://github.com/xataio/website/issues/138